requests-example.http

```diff
@@ -11,7 +11,7 @@ Accept: application/json;charset=utf-8
     "zip":70115,
     "description":"fabulous",
     "rent":1000.50,
-    "furniture_items":[
+    "furnitures":[
```

Comment:
furniture is the plural of furniture, and an item of furniture is the singular. not sure what's going on here with this rename